packages/mobile/package.json

```diff
@@ -69,31 +69,47 @@
     "@segment/analytics-react-native-adjust": "^1.4.3",
     "@segment/analytics-react-native-firebase": "^1.4.3",
     "@sentry/react-native": "^2.1.0",
+    "@tradle/react-native-http": "^2.0.1",
     "@ungap/url-search-params": "^0.1.2",
+    "@walletconnect/client": "2.0.0-alpha.31",
     "Base64": "^1.0.1",
     "apollo-boost": "^0.4.9",
     "apollo-cache-persist": "^0.1.1",
+    "assert": "^1.5.0",
     "bignumber.js": "^9.0.0",
     "bn.js": "4.11.8",
+    "browserify-zlib": "^0.1.4",
+    "buffer": "^4.9.2",
```

Comment:
Looks like lots of new dependencies here. Are all these necessary? I could be mistaken (or things could have changed) but I believe that polyfills for common node libs like buffer were already included in the wallet: https://github.com/celo-org/wallet/blob/main/packages/mobile/metro.config.js#L2

Reply:
You're right they're not. They're from when I was messing around rn-nodeify